packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/RelationFromManyFieldDisplay.tsx

```diff
@@ -45,19 +50,40 @@ export const RelationFromManyFieldDisplay = () => {
         : CoreObjectNameSingular.Task;
 
     const relationFieldName = fieldName === 'noteTargets' ? 'note' : 'task';
+    const formattedRecords = fieldValue.map((record) => {
+      if (!isDefined(record)) {
+        return undefined;
+      }
+
+      if (!isDefined(record[relationFieldName])) {
+        return {
+          ...record,
+          [relationFieldName]: {
+            id: 'fallback-id',
+            title: pascalCase(relationFieldName),
+          },
+        };
+      }
 
+      return record;
+    });
     return (
       <ExpandableList isChipCountDisplayed={isFocused}>
-        {fieldValue
-          .map((record) =>
-            isDefined(record) && isDefined(record[relationFieldName]) ? (
+        {formattedRecords
+          .map((record) => {
+            if (!isDefined(record)) {
+              return undefined;
+            }
+
+            return (
               <RecordChip
                 key={record.id}
                 objectNameSingular={objectNameSingular}
                 record={record[relationFieldName]}
+                forceDisableClick={isReadOnly}
               />
-            ) : undefined,
```

Comment:
Probably stupid question, but why didn't you keep the `fieldValue.map` logic (in other terms why the need of `formattedRecords`) and simply shoot a `RecordChip` to substitute the `undefined` return ? Sounds easier DX to me no ?